composer.json

```diff
@@ -13,7 +13,8 @@
     "ptachoire/process-builder-chain": "~1.2.0",
     "guzzlehttp/guzzle": "~6.2.0",
     "rudloff/rtmpdump-bin": "~2.3",
-    "aura/session": "~2.1.0"
+    "aura/session": "~2.1.0",
+    "smarty-gettext/smarty-gettext": "~1.2.0"
```

Comment:
You should use smarty-gettext 1.5.1.
1.2.0 does not work correctly with the way Smarty is set up in Alltube.